src/packs/checker.rs

```diff
@@ -147,7 +143,6 @@ struct CheckAllBuilder<'a> {
 }
 
 struct FoundViolations {
-    checkers: Vec<Box<dyn CheckerInterface + Send + Sync>>,
```

Comment:
Move, pushing this responsibility to make strict a property of the identifier simplifies things quite a bit!